padding/padding_test.go

```diff
@@ -77,9 +86,94 @@ func TestPaddingWriter(t *testing.T) {
 }
 
 func TestPaddingString(t *testing.T) {
+	t.Parallel()
+
 	actual := String("foobar", 10)
 	expected := "foobar    "
 	if actual != expected {
 		t.Errorf("expected:\n\n`%s`\n\nActual Output:\n\n`%s`", expected, actual)
 	}
 }
+
+func BenchmarkPaddingString(b *testing.B) {
+	var actual string
+	expected := "foobar    "
+
+	b.ReportAllocs()
+	b.ResetTimer()
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			actual = String("foobar", 10)
+		}
+	})
+
+	if actual != expected {
```

Comment:
Same as with #11: we should either make this concurrency-safe and check the result within the `for` loop, or drop the check entirely and only focus on the part we actually want to benchmark.

Reply:
Done.